css/components/_form-questions.scss

```diff
@@ -0,0 +1,149 @@
+form {
```

Comment:
Use classes instead of targeting elements. This selector is very specific and could affect any form of the website.